lib/puppet/transport/philips_hue.rb

```diff
@@ -0,0 +1,57 @@
+require 'faraday'
+require 'uri'
+require 'hocon'
+require 'hocon/config_syntax'
+require 'puppet/util/network_device/base'
+require 'json'
+
+module Puppet::Transport
+  # A basic device class, that reads its configuration from the provided URL.
+  # The URL has to be a local file URL.
```

Comment:
If we are now using a connection_info hash, are we using file URLs?

Reply:
This hasn't been changed.

Reply:
Yeah, these comments can be remote

Reply:
removed